lib/ansible/plugins/connection/winrm.py

```diff
@@ -76,7 +77,7 @@ def set_host_overrides(self, host):
         '''
         Override WinRM-specific options from host variables.
         '''
-        host_vars = host.get_vars()
+        host_vars = combine_vars(host.get_group_vars(), host.get_vars())
```

Comment:
hmm, forgot about that 'feature', still think that a generic function in play_context that gets ansible_<pluginname>_* vars injected might be a better approach

@jimi-c @abadger ?